src/redux/reducer.js

```diff
@@ -75,21 +51,21 @@ const initialState = {
     cases: [],
     selectedCase: null,
     selectedFile: null,
-    useName: getLocalStorageUseName(),
+    useName: true,
     user: null,
-    centerLabel: getLocalStorageCenterLabel(),
-    diagonalLabel: getLocalStorageDiagonalLabel(),
-    lineFullPath: getLocalStorageLineFullPath(),
-    lineParallelPath: getLocalStorageLineParallelPath(),
-    lineFlowMode: getLocalStorageLineFlowMode(),
-    lineFlowColorMode: getLocalStorageLineFlowColorMode(),
-    lineFlowAlertThreshold: getLocalStorageLineFlowAlertThreshold(),
+    centerLabel: false,
```

Comment:
It is not possible to use a dictionary ?

Reply:
what do you mean ?